pagopa/io-app
# feat: [IOBP-76] Add IDPay transaction cancellation

Depends on https://github.com/pagopa/io-app/pull/4695

## Short description
This PR adds the IDPay transaction cancellation on "deny" button press during the authorization process

## List of changes proposed in this pull request
- Added required locale keys
- [ts/features/idpay/payment/screens/IDPayPaymentAuthorizationScreen.tsx](https://github.com/pagopa/io-app/pull/4696/files#diff-13523f24396fea11f6aa4e54f4550223619cc436304b69e0de32772019e1089a): added `isLoading` prop to cancel button when cancelling payment authorization
- [ts/features/idpay/payment/screens/IDPayPaymentResultScreen.tsx](https://github.com/pagopa/io-app/pull/4696/files#diff-13a0efdb2dfe2a5ba04e61cae49a65059b02ec0790ea78f640cbd80b601d5580): added cancelled screen component
- Added tests

## How to test
Tests should solve correctly.
With the `io-dev-server`, use transaction code `00000000` and try to cancel the authorization.
